src/modules/local_storage.js

```diff
@@ -0,0 +1,64 @@
+import createElements from './new_todo.js';
+
+const todoInput = document.querySelector('.todo-input');
+
+const existOrNot = () => {
+  let items;
+  if (localStorage.getItem('items') === null) {
+    items = [];
+  } else {
+    items = JSON.parse(localStorage.getItem('items'));
+  }
+  return items;
+};
+
+const saveTodoInLocalStorage = (item) => {
+  const items = existOrNot();
+
+  items.push(item);
+
+  localStorage.setItem('items', JSON.stringify(items));
+};
+
+export function addTodo(e) {
+  e.preventDefault();
+
+  createElements(todoInput.value);
+
+  saveTodoInLocalStorage(todoInput.value);
+
+  todoInput.value = '';
+}
```

Comment:
- [ ] [REQUIRED] From the image below what you have is an array of strings, but what you should have is an array of objects. Each of the task object should have `description` (string), `completed` (boolean), and `index` (number) properties. Kindly refactor accordingly.

  ![Screenshot from 2022-03-31 19-52-35](https://user-images.githubusercontent.com/58692590/161130603-a4e41c57-8c66-4655-96c2-fd5becd2f05b.png)

- [ ] [REQUIRED] When a task is added the `completed` property should be set to `false` and `index` property should be set to the new array length.

  ```
  [
    {description: 'first task', completed: false, index: 1}
  ]
  ```